src/Features/Core/Portable/ConvertAutoPropertyToFullProperty/AbstractConvertAutoPropertyToFullPropertyCodeRefactoringProvider.cs

```diff
@@ -123,12 +123,12 @@ internal static bool IsValidAutoProperty(IPropertySymbol propertySymbol)
             var typeDeclaration = propertySymbol.ContainingType.DeclaringSyntaxReferences;
             foreach (var td in typeDeclaration)
             {
-                var block = GetTypeBlock(await td.GetSyntaxAsync(cancellationToken).ConfigureAwait(false));
-                if (property.Ancestors().Contains(block))
+                var typeBlock = GetTypeBlock(await td.GetSyntaxAsync(cancellationToken).ConfigureAwait(false));
+                if (property.Ancestors().Contains(typeBlock))
                 {
-                    editor.ReplaceNode(block, (currentTypeDecl, _)
-                        => codeGenerator.AddField(currentTypeDecl, newField, info, cancellationToken)
-                        .WithAdditionalAnnotations(Formatter.Annotation));
+                    editor.ReplaceNode(
+                        typeBlock,
+                        (currentTypeDeclaration, _) => codeGenerator.AddField(currentTypeDeclaration, newField, info, cancellationToken));
```

Comment:
this is the core change.  we were attaching the formatting annotation to the entire class.  Def not something we ever want to do.

Reply:
Does the generator attach it to the new field?

Reply:
yes :)